ext/mbstring/mbstring.c

```diff
@@ -1,20 +1,20 @@
 /*
-   +----------------------------------------------------------------------+
-   | Copyright (c) The PHP Group                                          |
-   +----------------------------------------------------------------------+
-   | This source file is subject to version 3.01 of the PHP license,      |
-   | that is bundled with this package in the file LICENSE, and is        |
-   | available through the world-wide-web at the following url:           |
-   | http://www.php.net/license/3_01.txt                                  |
-   | If you did not receive a copy of the PHP license and are unable to   |
-   | obtain it through the world-wide-web, please send a note to          |
-   | license@php.net so we can mail you a copy immediately.               |
-   +----------------------------------------------------------------------+
-   | Author: Tsukada Takuya <tsukada@fminn.nagano.nagano.jp>              |
-   |         Rui Hirokawa <hirokawa@php.net>                              |
-   |         Hironori Sato <satoh@jpnnet.com>                             |
-   |         Shigeru Kanemoto <sgk@happysize.co.jp>                       |
-   +----------------------------------------------------------------------+
+	 +----------------------------------------------------------------------+
+	 | Copyright (c) The PHP Group                                          |
+	 +----------------------------------------------------------------------+
+	 | This source file is subject to version 3.01 of the PHP license,      |
+	 | that is bundled with this package in the file LICENSE, and is        |
+	 | available through the world-wide-web at the following url:           |
+	 | http://www.php.net/license/3_01.txt                                  |
+	 | If you did not receive a copy of the PHP license and are unable to   |
+	 | obtain it through the world-wide-web, please send a note to          |
+	 | license@php.net so we can mail you a copy immediately.               |
+	 +----------------------------------------------------------------------+
+	 | Author: Tsukada Takuya <tsukada@fminn.nagano.nagano.jp>              |
+	 |         Rui Hirokawa <hirokawa@php.net>                              |
+	 |         Hironori Sato <satoh@jpnnet.com>                             |
+	 |         Shigeru Kanemoto <sgk@happysize.co.jp>                       |
+	 +----------------------------------------------------------------------+
```

Comment:
I have no idea where the formatting comes from, but these headers always look the same, using a three space indent. I'd suggest to leave it at that.

Reply:
OK, will do!